controllers/chat/initChatFromPost.js

```diff
@@ -99,7 +99,7 @@ const initChatFromPost = async (req, res) => {
 
       await ChatAnonUserInfo.create({
         channel_id: newChannel.id,
-        my_anon_user_id: userModel.user_id,
+        my_anon_user_id: targetUserModel.user_id,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `userModel.user_id` to `targetUserModel.user_id` for the `my_anon_user_id` field might cause issues if the intention was to keep track of the user who initiated the chat. If `targetUserModel` is the recipient of the chat, then this change would incorrectly assign the recipient as the initiator. Please ensure that this change aligns with the intended functionality.
```diff
- my_anon_user_id: targetUserModel.user_id,
+ my_anon_user_id: userModel.user_id,
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->